packages/bpk-component-select/src/bpk-select.scss

```diff
@@ -41,71 +41,65 @@
     }
   }
 
-  &--borderless {
-    height: auto;
-    padding: 0;
-    border: transparent;
+  &--with-image {
+    padding-left: $bpk-spacing-xl;
 
-    @include bpk-rtl {
-      padding: 0;
+    @media screen\0 {
```

Comment:
These are to combat the IE fixes located in the `bpk-select` mixin -> https://github.com/Skyscanner/backpack/blob/d01d46016361a1d7f5cc28bdd27811341428a367/packages/bpk-mixins/src/mixins/_forms.scss#L321